main/consts.js

```diff
@@ -17,31 +14,3 @@ module.exports = Object.freeze({
 
   ELECTRON_VERSION: process.versions.electron
 })
-
```

Comment:
It would be nice if we could make `core` place Saturn files in the exact same location as before, so that the job count from before gets used. Problem: `core`'s Saturn places files in `$CACHE_ROOT/modules/saturn-l2-node`, while Desktop puts it in `$CACHE_HOME/saturn`. @bajtos what do you think?

Reply:
> It would be nice if we could make core place Saturn files in the exact same location as before, so that the job count from before gets used. 

💯 

If the upgrade resets the job counter, then IMO we need to explain to users what happened and why.

Can we implement data migration that will move Saturn state files from `$CACHE_HOME/saturn` to `$CACHE_ROOT/modules/saturn-l2-node` as part of Station Desktop startup, but before the Station Core & Saturn L2 module is started?

This can be a one-off thing, or we can look into adding some framework for writing migrations.

For example, in the former case, we can check for the presence of the old state directory and the new state directory. If we see the old one but not the new one, then we can move the old one into the new location.

Reply:
Good idea to write a migration! Should be simple enough using `fs`.

One question I have is though how the migration tool will know where to move files. If we want to remove the logic for figuring out where to store everything per OS (https://github.com/filecoin-station/core#common-configuration), then only `core` knows where to place the files.

@bajtos what do you think about adding a new command `$ core config` or `$ core paths`, which lets you read its default values? Then we can migrate to `${exec('core config CACHE_ROOT').stdout}/modules/saturn-l2-node`.

Reply:
We decided in a call to keep the this path logic here, since Desktop needs to own its own paths and tell `core` where to place its files.